jupyter/nmm-game.ipynb

```diff
@@ -41,7 +47,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Player Phase"
+    "## Spieler\n",
+    "\n",
+    "Zunächst soll die Menge der Spieler definiert werden. Da Mühle mit zwei Spielern gespielt wird, die jeweils weiße oder schwarze Steine legen, werden die Spieler äquivalent benannt.\n",
+    "* Der beginnende Spieler weiß wird als `w` dargestellt,\n",
```

Comment:
Hmm ist eigentlich falsch, da weiß nicht unbedingt anfangen muss.

Reply:
_Der Spieler mit den helleren Spielsteinen (in der Regel Weiss) beginnt die Partie._ [Offizielle Tunierregeln](http://www.muehlespiel.eu/images/pdf/WMD_Turnierreglement.pdf)

Reply:
Wieder was gelernt ^^